src/helpers/queries.ts

```diff
@@ -506,3 +506,19 @@ export const SPACE_QUERY = gql`
     }
   }
 `;
+
+export const DELEGATE_VOTES_AND_PROPOSALS = gql`
+  query VotesAndProposals($delegates: [String]!, $space: String!) {
+    votes(where: { voter_in: $delegates, space: $space }) {
+      created
+      voter
+      choice
+      vp
+    }
+    proposals(where: { author_in: $delegates, space: $space }) {
+      created
+      author
+      title
```

Comment:
What's the reason for including these fields? Maybe created is enough? I know I included a few fields in the pitch but tbh it was a bit random. Don't think it has noticeable effects on performance, so no big deal anyway

Reply:
We will need these fields later when we show the detailed list of votes and proposals.